pkg/simple/client/auditing/elasticsearch/options.go

```diff
@@ -47,6 +48,7 @@ func (s *Options) Validate() []error {
 }
 
 func (s *Options) AddFlags(fs *pflag.FlagSet, c *Options) {
+	fs.BoolVar(&s.Enabled, "auditing-enabled", c.Enabled, "Dose auditing enabled")
```

Comment:
Typo `Does`,  I think we can following description.
```suggestion
	fs.BoolVar(&s.Enabled, "auditing-enabled", c.Enabled, "Enable auditing component or not.")
```